### PR TITLE
feat(reference): Adds host configuration docs

### PR DESCRIPTION
**Pull request by thomastaylor312**

### CODE REVIEW

File: docs/reference/host-config.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+| `--otel-traces-exporter`        | `OTEL_TRACES_EXPORTER`                                   | N/A       | Specifies which exporter to use for traces (Only "otlp" is currently supported)                                                                                                                                                                           | `--otel-traces`-exporter otlp                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (brooksmtownsend) [inline docs/reference/host-config.md]:
```suggestion
| `--otel-traces-exporter`        | `OTEL_TRACES_EXPORTER`                                   | N/A       | Specifies which exporter to use for traces (Only "otlp" is currently supported)                                                                                                                                                                           | `--otel-traces-exporter otlp`                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

File: docs/reference/host-config.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| `--cluster-seed`                | `WASMCLOUD_CLUSTER_SEED`                                 | N/A       | The seed key used by this host to sign all invocations.                                                                                                                                                                                                   | `--cluster-seed` SCAICHCEWFXMDS6AFL2DAGLXE6HMAL23CW2RRJUXJKJMN7VFP4W7XXLYZE                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (brooksmtownsend) [inline docs/reference/host-config.md]:
```suggestion
| `--cluster-seed`                | `WASMCLOUD_CLUSTER_SEED`                                 | N/A       | The seed key used by this host to sign all invocations.                                                                                                                                                                                                   | `--cluster-seed SCAICHCEWFXMDS6AFL2DAGLXE6HMAL23CW2RRJUXJKJMN7VFP4W7XXLYZE`                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
All looks good just requesting some changes that I'll take care of since Taylor is OOO

**Review by brooksmtownsend — APPROVED** [review]

**Comment by connorsmith256** [comment]
@brooksmtownsend https://github.com/wasmCloud/wasmcloud.com-dev/pull/218 superceded this one

**Pull request closed without merging**